test/system/010-images.bats

```diff
@@ -370,6 +370,13 @@ EOF
     run_podman --root $imstore/root rmi --all
 }
 
+@test "podman images with retry" {
+    run_podman pull -q --retry 4 --retry-delay "10s" $IMAGE
```

Comment:
I'm not seeing much point to this. However, I think something like this could be made to work?
```bash
    (iptables -A OUTPUT -m tcp -p tcp --dport 443 -j REJECT; sleep 10; iptables -D FIXME-FIGURE-OUT-SOME-WAY-To-IDENTIFY-AND-REMOVE-THAT-LAST-RULE) &
    run_podman ... retry ... pull quay.io/thisdoes/not-exist:really
    assert "$output" =~ "Failed, retrying in .* pinging.*connection refused"
    # quay returns 403 for 404
    assert "$output" =~ "unauthorized: access .* not authorized"
```

Reply:
Oh good, please never ever block port 443. This is run on developer systems and even in CI it could cause all kinds of havoc, I would guess cirrus agent or whatever this thing is called communicates via some https API so you may end up blocking that as well.

If want an actual test for this then why not setup a local registry and try to pull from that. We can start/stop the registry container as we like to simulate pull errors which would then even work rootless.

Reply:
Tried to add a registry and then pause the registry and do a pull, but the pull just hangs, so not sure if there is a good way to test.  PTAL